pages/en/lb4/Decorators.md

```diff
@@ -0,0 +1,379 @@
+  ) {}
+  // etc...
+}
+```
+
+For more information, see the [Dependency Injection](Dependency-Injection.htm) section under [LoopBack Core Concepts](Concepts.htm)
+
+## Authentication Decorator
+
+  Syntax: `@authenticate(strategyName: string, options?: Object)`
+
+  Marks a controller method as needing an authenticated user. 
+  This decorator requires a strategy name as a parameter.
+
+  Here's an example using 'BasicStrategy': to authenticate user in function `whoAmI`:
+
+  ```ts
+  // my-controller.ts
+  import { authenticate } from '@loopback/authentication';
+  import { inject } from '@loopback/context';
+
+  class MyController {
+    constructor(
+      @inject(BindingKeys.Authentication.CURRENT_USER)
+      private user: UserProfile,
+    ) {}
+
+    @authenticate('BasicStrategy')
+    async whoAmI() : Promise<string> {
+      return this.user.id;
+    }
+  }
+  ```
+
+## Repository Decorators
+
+  As a Domain-driven design concept, 
+  repository means a mediate between your domain object and data mapping layers
+  using a collection-like interface for accessing domain objects. 
+  In framework LoopBack, a domain object is usually a TypeScript/JavaScript Class instance, 
+  and a typical example of data mappting layer module could be a database's node.js driver.
+  
+  LoopBack repository encapsulates your TypeScript/JavaScript Class instance, 
+  and its methods that communicate with your database.
+  It is an interface to implement data persistency. 
+
+  Repository decorators are used for defining models(domain objects) for use with your chosen datasources, 
+  and the navigation strategies among models.
+
+  If you are not familiar with repository related concepts like Model, Entity, Datasource,  
+  please see LoopBack concept [Repositories](#Repositories.htm) to learn more. 
+  
+### Model Decorators
+
+  Model is a Class that LoopBack builds for you to organize the data that 
+  share same configurations and properties.
+  You can use model decorators to define a model and its properties.
+
+#### Model Decorator
+
+  Syntax: `@model(definition?: ModelDefinitionSyntax)`
+
+  Model decorator is a Class decorator.
+  In LoopBack 4, we inherit the model definition format from LoopBack 3,
+  you can find it in [Model definition JSON file](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html).
+  For usage examples, see [Define Models](Repositories.html#define-models)
+
+  *Please note we will elaborate more about model and model definition in #Model.htm,*
+  *and replace the link above with LB4 link*
+
+  By using a model decorator, you can define a model as your repository's metadata,
+  and resolve your repository with Legacy Juggler that defautly comes with a set of CRUD operations bound to your model.
+  A use case can be found in section [Repository decorator](#repository-decorator)
+
+  Or you also have the flexibility to define a model without being resolved with Legacy Juggler.
```

Comment:
`You can also define your own repositories without using legacy juggler, and use an ORM/ODM of your choice.`
(You can probably drop the 2nd line as well; feel free to copy-paste this comment. :P )

Reply:
wait...this is in section#model-decorator, so it's talking about how to define `Model` not `Repository` :)

Reply:
The reason why I explain "with-juggler" and "without-juggler" scenarios in model decorator is:

By using decorator `@repository`, you already choose legacy juggler.